Software/Rig/trial.h

```diff
@@ -0,0 +1,42 @@
+/* Set of constants that define the trial. Change these before uploading
+ * a sketch for each trial.
+ *
+ * NOTE: some of these should be `int` but had to be `long` during
+ * debugging for unclear reasons
+ *
+ */
+
+#ifndef TRIAL
```

Comment:
Some cleanup will make this more straightforward, but the real missing functionality is randomization